.github/workflows/coverage.yml

```diff
@@ -111,7 +111,7 @@ jobs:
           SLIDING_SYNC_PROXY_URL: "http://localhost:8118"
 
       - name: Upload to codecov.io
-        uses: codecov/codecov-action@v4
+         codecov/codecov-action@v4
```

Comment:
v4 doesn't support tokenless uploads anymore: https://github.com/codecov/codecov-action?tab=readme-ov-file#breaking-changes.

I added the token using the `CODECOV_TOKEN` name as described in: https://docs.codecov.com/docs/adding-the-codecov-token#github-actions.

Could you please add it as described over here: https://github.com/codecov/codecov-action?tab=readme-ov-file#usage.

Reply:
Wait I didn't actually change the action version. But strangely the `uses:` was removed here.

Reply:
Fixed

Reply:
Ah indeed.